app/assets/javascripts/application.js

```diff
@@ -13,4 +13,6 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
+//= require boilerstrap
```

Comment:
I was reading up on boilerstrap. According to the docs several key features are now deprecated. The maintainers suggest that you use the boltstrap library instead instead. http://html9responsiveboilerstrapjs.com